my-dsl.md

```diff
@@ -1,38 +1,40 @@
 # Language
-_What is the name of the language? Link the name to its webpage 
-(if appropriate)._
+Alda
+[Source Code](https://github.com/alda-lang/alda)
 
 
 # Domain
-_Describe the language's domain in five words._
+General Light-weight Music Programming Language
 
 
 # Computational model
-_What is the underlying computational model of this language? To answer this 
-question, provide a high-level description (no more than 100 words) of the 
-computation that occurs when someone executes a program in this language._
-
+When someone runs an alda script, their code is first converted into Clojure
+code which creates a map of the score. That is then used to produce sounds 
+that the alda code described. 
 
```

Comment:
This is indeed a pretty terse description of what seems like a super-interesting language! What do you mean by "map" of the score? How are parts of the language related to the "map"?